packages/kbn-calculate-width-from-char-count/src/calculate_width_from_char_count.ts

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+interface LIMITS {
+  paddingsWidth: number;
+  minWidth?: number;
+  avCharWidth: number;
+  maxWidth: number;
+}
+
+export const MAX_WIDTH = 550;
+const PADDINGS_WIDTH = 116;
+const AVERAGE_CHAR_WIDTH = 7;
+
+const defaultPanelWidths: LIMITS = {
+  maxWidth: MAX_WIDTH,
+  avCharWidth: AVERAGE_CHAR_WIDTH,
+  paddingsWidth: PADDINGS_WIDTH,
+};
+
+export function calculateWidthFromCharCount(
```

Comment:
can we expose also a `calculateWidthFromEntries` which computes the 
```ts
array.reduce(
        (acc, curr) => (acc > curr.label.length ? acc : curr.label.length)
```

and passes the result to this function?

I see that 99% of the code replacement is doing mostly this reduction bit.